model/stats/stats.go

```diff
@@ -0,0 +1,276 @@
+// Provides functions to generate and query pre-computed test and task statistics.
+// The statistics are aggregated per day and a combination of (test, task, variant, distro, project, requester) for
+// tests and a combination of (task, variant, distro,  project, requester) for tasks.
+// For tests intermediate hourly statistics are also stored to avoid some re-computation.
+package stats
+
+import (
+	"time"
+
+	"github.com/evergreen-ci/evergreen/db"
+	"github.com/mongodb/grip"
+	"github.com/pkg/errors"
+	"gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
+)
+
+const (
+	fourWeeks = 4 * 7 * 24 * time.Hour
```

Comment:
can you name this after what it is used for? so we can change the interval later?